spec/spec_helper.rb

```diff
@@ -25,6 +25,9 @@
 
 $LOAD_PATH << '../app/domain'
 
+# Add conjur-cli load path
```

Comment:
I don't think this comment is necessary as is but maybe you could mention why we have to do this?